trax/data/tf_inputs.py

```diff
@@ -1107,6 +1112,42 @@ def bert_double_sentence_preprocess(batch):
     yield value_vector, segment_embs, segment_embs, label, np.int64(1)
 
 
+def mask_random_tokens(batch, vocab_size=30522, masking_prob=0.15, merge_with_loss=True):
+  """
+  Follows standard procedure of masking in BERT:
+  `masking_prob` random tokens are selected out of which:
+    80% are replaced with [MASK] token
+    10% are replaced with random token
+    10% are leaved with original token unchanged
+
+  ids of the tokens are from original lowercase vocab
+  """
+
+  for token_ids, *row_rest in batch:
+    y = token_ids.copy()
+
+    is_special_token = np.logical_or(token_ids == 101, token_ids == 102) # CLS and SEP tokens
```

Comment:
Let's make the CLS and SEP token-ids arguments in this function -- these maybe different token ids for different vocabularies, so they should be settable in the argument to this function.

Reply:
Thank you! That's a very simple and clean solution. I knew these magic numbers were a problem, but somehow I thought only about reading these special ids from vocabulary which was problematic in some cases, so I decided to leave them to deal with them some other day. Your solution is obviously much better.

Reply:
I added all special ids as arguments for function